meshtastic/node.py

```
@@ -982,49 +1103,275 @@ def setURL(self, url: str, addOnly: bool = False) -> None:
+                    last_rollback_error: Exception | None = None
+                    for rollback_admin_index in rollback_admin_indexes:
+                        try:
+                            self._write_channel_snapshot(
+                                rollback_channel,
+                                adminIndex=rollback_admin_index,
+                            )
+                            rollback_succeeded = True
+                            break
+                        # Best-effort rollback path; keep attempting remaining steps.
+                        except (
+                            Exception
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)